lib/srv/desktop/README.md

```diff
@@ -15,183 +15,55 @@ See [this appendix](#appendix-virtualbox-notes) if using VirtualBox.
 
 #### AD DS
 
-First, we need to install Active Directory Domain Services (AD DS). From Server
-Manager, in the top-right select `Manage > Add Roles and Features`.
-
-In the wizard, select:
-- Before you Begin: click `Next`
-- Installation Type: `Role-based or feature-based installation`
-- Server Selection: click `Next` (should be only one server - current one)
-- Server Roles: select `Active Directory Domain Services`, in the popup click `Add Features`
-- Features: click `Next`
-- AD DS: click `Next`
-- Confirmation: click `Install`, wait for completion
-- Results: click `Promote this server to a domain controller` blue link
-
-Another wizard (AD DS configuration) will open:
-- Deployment Configuration: select `Add a new forest`, type in the Root domain
-  name. Any DNS-like name will work, for example `example.com`, **write down
-  this name for later**. Click `Next`.
-- Domain Controller Options: type in a `Directory Services Restore Mode (DSRM)
-  password` and click `Next`.
-- DNS Options: click `Next`
-- Additional Options: wait for the NetBIOS name to be generated, **write it
-  down for later**, click `Next.`
-- Paths: click `Next`.
-- Review Options: click `Next`.
-- Prerequisites Check: wait for the check to pass and click `Install`.
-- Results: after ~10s, VM will restart
+First, we need to install Active Directory Domain Services (AD DS). Save the following file as `domain-controller.ps1`,
+replacing `$domain` with your desired domain name and `$adminPassword` with your Administrator account's password.
+
+```powershell
+$domain = 'teleport.dev'
+$adminPassword = 'teleport123!!'
```

Comment:
this will result in many, many installations using the hardcode password from the samples. Can you instead use the powershell script to auto generate the password and not mention it here? E.g. $(uuid) > password.ldap and then instruct to copy the contents or the file?

Reply:
This file is primarily for internal use, the intention isn't to share with customers how to set up an Active Directory domain. I mostly use it for reference when spinning up a new VM for testing.

There is a [place](https://github.com/gravitational/teleport/blob/87555836385ce18c22aa89dd2402c48556dc61b1/docs/pages/desktop-access/getting-started.mdx#L42) in the Getting Started where we require a password, in which powershell prompts the user to enter the password. 

An issue with this is that there's no prompt for the user to type the password again for confirmation, so if they fat finger something they might have problems. I'll switch it to your solution, since saving the password to a file also fits in neatly with the user needing to point to that file in the `teleport.yaml`.

Reply:
would it still be possible to remove the hardcode from this snippet? I'm afraid folks will copy paste it and we would get some breaches this way

Reply:
Yeah good call.

Reply:
@klizhentas actually that script wasn't setting the password, it was asking for the user's existing password. 

I've changed it so that now the script pauses and waits for the user to enter their password using `(Read-Host "Enter Your Password" -AsSecureString)`: this way it will never show up in plaintext, either in the script or in PowerShell or other logs.